examples/twittermap/web/public/javascripts/timeseriescache/services.js

```diff
@@ -0,0 +1,149 @@
+/*
+ * This module caches time-series histogram data. The common services module communicates with
+ * middle ware and the time-series cache.
+ */
+'use strict';
```

Comment:
What's this "strict"?

Reply:
strict mode is used mainly to make sure variables are declared before use. as in `mapresultcache/services.js`.

Reply:
Got it.